Gruntfile.js

```diff
@@ -216,17 +233,25 @@ module.exports = function(grunt) {
       grunt.fail.fatal('You must specify a release type. i.e. grunt release:prerelease');
     }
 
-    grunt.task.run([
-      'prompt:gh-release',
-      'build',
-      'test:unit',
-      'copy:dist',
+    var dryRun = grunt.option('dry-run');
+
+    var tasks = ['prompt:gh-release'];
+
+    if (dryRun) {
+      grunt.log.ok('Skipping build, test:unit and copy:dist tasks in dry-run');
```

Comment:
We might want to keep build and test in there, since they don't modify any files actually in git? What do you think?